dask-metrics/conda.recipe/build.sh

```diff
@@ -0,0 +1 @@
+$PYTHON /raid/thester/dask-metrics/setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
Could this be updated to use a path relative to the current folder rather than an absolute path